docs/edgeql/ddl/links.rst

```diff
@@ -272,7 +279,7 @@ The following subcommands are allowed in the ``ALTER LINK`` block:
     Make the link *required*.
 
 :eql:synopsis:`DROP REQUIRED`
-    Make the link no longer *required*.
+    Make the link *optional*, i.e. no longer *required*.
```

Comment:
We have to fix this in a follow up PR. @elprans

Reply:
Yeah, we talked about it a few hours back.

Reply:
I do remember that. I'm reinforcing the fact that we need to fix this before a3.